files/mrtg.sh

```diff
@@ -52,7 +55,12 @@ sleep 2
 env LANG=C /usr/bin/mrtg ${MRTGCFG}
 sleep 2
 env LANG=C /usr/bin/mrtg ${MRTGCFG}
-/usr/bin/indexmaker --columns=1 ${MRTGCFG} -rrdviewer=${PATHPREFIX}/cgi-bin/14all.cgi --icondir=/ --prefix=${PATHPREFIX}/ >> ${WEBDIR}/index.html
+
+# Only run indexmaker if the index file does not exist. Without this, each container restart the index will be regenerated and appended to the existing file.
+if [ ! -e "${WEBDIR}/index.html" ]; then
+   /usr/bin/indexmaker --columns=1 ${MRTGCFG} -rrdviewer=${PATHPREFIX}/cgi-bin/14all.cgi --icondir=/ --prefix=${PATHPREFIX}/ >> ${WEBDIR}/index.html
```

Comment:
@TweakM I remember having a reason to re-run the `indexmaker` on every reboot. I won't be able to validate it now, but if I'm not mistaken, we need this to account for new assets added to the list.

If that is the case, we may then regenerate the index on each run and overwrite the file. Can you please, validate?

Reply:
@fboaventura You are right, I did not take the use case of adding new assets into account. Due to the append bug it still needs a fix though. I'll see if I can create an update that satisfies both use cases (my use case being a manually modified index.html and/or a manually modified cfg file).

Reply:
@TweakM replacing the >> by > fixes the ever-increasing index.html

I can think of two quick paths: 1. Storing the number of assets in a file and checking this file on each run to decide if it will execute indexmaker or not, and 2. Have a variable to decide if it will be updated or not.

Depending on the customization depth, we may have new variables to go with indexmaker [options](https://manpages.ubuntu.com/manpages/xenial/man1/indexmaker.1.html). 

Would this be enough? If not, do you mind sharing some of your customizations?

Reply:
@fboaventura I have implemented the second path (a variable that determines if the `index.html` should be regenerated or not). I have also included an environment variable to specify extra `indexmaker ` options. The documentation has also been updated with my latest [commit](https://github.com/fboaventura/dckr-mrtg/pull/13/commits/5d15dcf85f6dd45d9f724096b2e4b4055e544013). My customizations are manual edits to the `index.html` that cannot be generated by `indexmaker`.
Hopefully this satisfies all your questions and requests.